components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeRowReader.java

```
@@ -107,6 +151,53 @@ public Instant getCurrentTimestamp() {
         return Instant.now();
     }
 
+    private boolean validateResultSet(ResultSet rs) throws IOException{
+        if (rs == null) {
+            return false;
+        }
+        try {
+            ResultSetMetaData rsMetadata = rs.getMetaData();
+
+            if (CUD_RESULT_SET_COLUMN_NAMES.contains(rsMetadata.getColumnName(1))) {
+                result.totalCount++;
+                result.successCount++;
+                return false;
+            }
+            return true;
+        } catch (SQLException e) {
+            if(dieOnError) {
+                throw new IOException(e);
+            }
```

Could you add some log message here? 
It is not good practice to ignore exception